Android/app/src/main/java/app/intra/DnsResolverUdpToHttps.java

```diff
@@ -87,9 +88,14 @@ static void processQuery(ServerConnection serverConnection, DnsUdpQuery query,
     }
 
     // Writes |dnsRequestId| to |dnsResponse|'s ID header.
-    private void writeRequestIdToDnsResponse(byte[] dnsResponse, short dnsRequestId) {
+    private boolean writeRequestIdToDnsResponse(byte[] dnsResponse, short dnsRequestId) {
```

Comment:
Do you see any advantages in returning a boolean instead of throwing the exception?

Reply:
That seems like a good approach.  Done.